### DISCUSSION TITLE
My benefits not available at approval status for 4 days

### DISCUSSION THREAD

**Discussion by JJLi0427** [community/community#148160] [GitHub Education] [answered]
### Hi everyone,

![image](https://github.com/user-attachments/assets/05d723bf-7d24-48e2-8357-9eaae0600c30)
I haven't receive any email and my github student pack not available, although it show I was approved in Dec 27

**Comment by msafeerhassan** [comment]
Same here! Please assist us.

**Comment by Olayiwolaaa** [comment]
I got approved like a day or two. My question to you is, Do you really need a mail?
Maybe try checking spam or reach out to github.

It should look like this : 
![image](https://github.com/user-attachments/assets/0cace31c-efeb-499e-bda9-1efebb6730a1)

**Reply by msafeerhassan** [reply]
I haven't gotten this. What should I do now?

**Reply by JJLi0427** [reply]
I didn't receive any email like you, and student pack benefits is not available to me. I read the FAQ it tells me that ask for help at there. Usually we will receive the email within three days.
![image](https://github.com/user-attachments/assets/36a3bc7f-bc22-47c7-b788-07e625f7bbe4)

**Comment by davevad93** [comment, marked as answer]
Hi @JJLi0427 ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @JJLi0427 was approved and your benefits are now available.`

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #145640, #145995

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by JJLi0427** [reply]
Thank you!
 I found I was approved today,  the status label turn purple now.

**Comment by samus-aran** [comment]
I'll be closing this discussion due to lack of recent activity. We can confirm that you have since been successful - enjoy your journey through GitHub Education.
